Feature/docs (#273)
infinum/eightshift-docs: website/forms/features/country-list.md

title: Country list
---

> We use [DataHub](https://datahub.io/core/country-list) as database for providing the list of countries. Eightshift forms are not  biased towards any country and has no political agenda. We use this list because it is the most complete and up to date list of countries.

Reviewer: ```suggestion
[DataHub](https://datahub.io/core/country-list) is the primary country database in Eightshift forms, as it's the most complete and up-to-date list of countries. This plugin is not biased towards any country, nor does it have any political agenda. Please forward any country list-specific feedback towards DataHub.
```

> All releases are listed [here](https://datahub.io/core/country-list) and you can find our list [here](https://github.com/infinum/eightshift-forms/blob/develop/data/country/manifest.json).

Reviewer: ```suggestion
All releases are listed [here](https://datahub.io/core/country-list), the latest version of the country list in Forms is available [here](https://github.com/infinum/eightshift-forms/blob/develop/data/country/manifest.json).
```

![Country list screen](/img/forms/country.webp)

## Where is the country list used?

> We use the country list in the following features:

Reviewer: ```suggestion
Country list provides data for the following features:
```

> * country block
> * phone blocks
> * geolocation features

Reviewer: ```suggestion
* _Country picker_ form field
* _Phone number_ form field
* Geolocation features
```

## How to add a new country or create a new list?

> If you want to add a new country or create a new list, you can do so by adding a custom filter in your project. You can find more details on how to do that [here](/forms/php/filters/block/country/alternative-data-set).

Reviewer: ```suggestion
To add a country or create a new list, you can use a filter that we provide. For more details on how to do that check [this page](/forms/php/filters/block/country/alternative-data-set).
```
